packages/xo-web/src/xo-app/vm/tab-backups.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'intl'
+import addSubscriptions from 'add-subscriptions'
+import constructQueryString from 'construct-query-string'
+import decorate from 'apply-decorators'
+import React from 'react'
+import { adminOnly } from 'utils'
+import { createPredicate } from 'value-matcher'
+import { injectState, provideState } from 'reaclette'
+import { some } from 'lodash'
+import { subscribeJobs } from 'xo'
+
+import LegacyOverview from '../backup/overview-legacy'
+import { JobsTable } from '../backup/overview'
+
+const ACTIONS = [
+  {
+    handler: (selectedJobs, { goTo }) =>
+      goTo({
+        pathname: '/backup/overview',
+        query: { s: `id: |(${selectedJobs.map(_ => _.id)})` },
+      }),
```

Comment:
Does it have to be an action? I thought maybe it could just be a link above the table and it wouldn't matter which backups are selected.